src/marshaller/HipieDDL.js

```diff
@@ -894,11 +900,17 @@
                     updatedViz.getInputVisualizations().forEach(function (inViz, idx) {
                         if (inViz._eventValues) {
                             for (var key in inViz._eventValues) {
+                                var changed = inViz === context;
```

Comment:
I'm really not sure what's going on here. This seems like a strange place to compare these two objects?